5g-control-plane/templates/bin/_upf-adapter-run.sh.tpl

```diff
@@ -14,4 +14,4 @@ cd /aether
 
 cat config/config.yaml
 
-GOTRACEBACK=crash ./upfadapter config/config.yaml
+GOTRACEBACK=crash ./upfadapter -config config/config.yaml
```

Comment:
As far as I read in the code, I do not see the `config/config.yaml` being actually used anywhere. Or am I missing something? Moreover, because of this, we are not able to `set` the log level for this component. That is, where is the `-config` and/or the `config/config.yaml` being used in the `upfadapter` code?

Reply:
I shared Helm chart changes first...I am testing upf adapter changes and once confident then I can raise that PR. Helm chart merge will not break anything.